src/commands/project/get-display-name.ts

```diff
@@ -0,0 +1,62 @@
+/* eslint-disable @typescript-eslint/no-use-before-define */
+/* eslint-disable @typescript-eslint/camelcase */
+import commander, { CommandOptions } from "commander";
+import * as fs from "fs";
+import { read as readBedrockYaml } from "../../lib/bedrockYaml";
+import { build as buildCmd, exit as exitCmd } from "../../lib/commandBuilder";
+import { logger } from "../../logger";
+import decorator from "./get-display-name.decorator.json";
+
+/**
+ * Executes the command, can all exit function with 0 or 1
+ * when command completed successfully or failed respectively.
+ *
+ * @param opts validated option values
+ * @param exitFn exit function
+ */
+export const execute = async (
+  opts: CommandOptions,
+  exitFn: (status: number) => Promise<void>
+): Promise<void> => {
+  let currentDirectory = process.cwd() + "/bedrock.yaml";
+  try {
+    // First, find a bedrock.yaml file in this directory and if not, iterate by
+    // one directory up.
+    while (!fs.existsSync(currentDirectory)) {
+      const split = currentDirectory.split("/");
+      if (split.length < 2) {
+        break;
+      }
+      currentDirectory = currentDirectory.replace(
+        split[split.length - 2] + "/",
+        ""
+      );
+    }
+    currentDirectory = currentDirectory.replace("/bedrock.yaml", "");
```

Comment:
thanks @dennisseah, I ended up changing the implementation quite a bit because the command should be ran from the root folder, and not service folder, that simplified the code. Please take a look again.